plugins/usbdmx/DMXCProjectsNodleU1Factory.cpp

```diff
@@ -30,14 +30,25 @@ namespace ola {
 namespace plugin {
 namespace usbdmx {
 
-const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID = 0x16d0;
-const uint16_t DMXCProjectsNodleU1Factory::PRODUCT_ID = 0x0830;
+const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID_1 = 0x16d0;
```

Comment:
I think at a minimum it would be good if these were named, rather than just some arbitrary numbers.

Reply:
I wonder what should happen about the name of the class/name(s) reported in OLA too, it seems slightly confusing if the other devices all show up as Nodle. There are a few possible ways to get round that, or others may feel it's not necessary ( @nomis52 ?).

Reply:
I agree, it might not be intuitive. On the other hand once you know it, you shouldn't really mind?

Reply:
Fair enough, unless Simon feels strongly, I'm willing to merge and leave it as a potential future enhancement (support with a wonky name is better than no support). Would you mind adding a TODO somewhere around the relevant area (probably where the VIDs/PIDs are defined noting that they current all appear with the same device name?

Reply:
This has now been done.